OFEP-flag-eval-wire-protocol.md

```diff
@@ -0,0 +1,55 @@
+## Flag Evaluation Wire Protocol
+
+## State: DRAFTING
+
+Create a wire protocol for RPC-based flag evaluation compliant with the OpenFeature specification.
+
+## Background
+
+OpenFeature offers a vendor-agnostic SDK that works across various open source and commercial offerings.
+Developers can choose the provider they want to use and include it in their codebase.
+This works well when a high-quality provider exists, and it's acceptable to include a vendor-specific dependency.
+However, that's not always the case.
+For example, many open source applications could benefit from having a standardized means of including feature flags.
+A flag evaluation wire protocol would allow developers to choose the architecture that best fits their needs.
+
+## In-process vs Remote Evaluation
+
+Many feature flag vendors offer SDKs that perform flag evaluation in-process.
+This provides fast and consistent flag evaluations with no additional deployment complexities.
+Remote evaluation logic can be written in a single language and be used interchangeably with any system complying with the wire protocol.
+That makes it ideal for open source projects that need a vendor-agnostic solution for feature flags.
+
+## Action items
+
+- Review the [existing proto schema](https://github.com/open-feature/schemas/blob/main/protobuf/schema/v1/schema.proto) to identify ways to make it provider agnostic.
```

Comment:
My immediate reaction to gRPC **only** was somewhat negative, since this restricts implementations somewhat. Even though clients can get flexibility with Web gRPC it's not shared in the server-side.

As a **wire protocol** I'm not entirely opposed.

But I would be interested in seeing OpenAPI definitions as part of this as well, unless there are significant issues with that approach.